packages/table/src/cell/formats/jsonFormat.tsx

```diff
@@ -31,10 +31,11 @@ export interface IJSONFormatProps extends ITruncatedFormatProps {
 
 export class JSONFormat extends React.Component<IJSONFormatProps, {}> {
     public static defaultProps: IJSONFormatProps = {
-        detectTruncation: true,
+        detectTruncation: false,
```

Comment:
Can delete this line and fall back to the default props defined in `TruncatedFormat`.

Reply:
Wouldn't this be true for `showPopover` as well? it's default here is `showPopover: TruncatedPopoverMode.WHEN_TRUNCATED,` which is the default